app/assets/stylesheets/pages/_my_bookings.scss

```diff
@@ -1,6 +1,6 @@
 .booking{
   margin-bottom: 25px;
-  img {
+  .planet-img {
```

Comment:
tu peux mettre un screenshot du rendu sur la view de booking svp ?

Reply:
![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/35280912/99639199-9d989800-2a47-11eb-946e-576a2cf69042.png)

Reply:
ok j'aimerais que tu mettes l'avatar par défaut de PJ, je te l'envoie sur slack

Reply:
done in commit #4